scrapers/napa/scrape.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+
+import concurrent.futures
+import gzip
+import csv
+import os
+import time
+
+import requests
+
+CONNECTIONS = 20
+PARCEL_LOOKUP_URL = 'https://common2.mptsweb.com/MBC/napa/tax/main/%s/2020/0000'
+PARCEL_SOURCE_FILE = '/home/alin/Downloads/napa/parcels_public.csv'
+OUTPUT_DIR = '/home/alin/code/prop13/scrapers/napa/scrape_output'
```

Comment:
Same statement as above about putting the files in a canonical location for repeatability.